OptimizelySDK/ProjectConfig.cs

```diff
@@ -551,5 +558,21 @@ public Rollout GetRolloutFromId(string rolloutId)
             ErrorHandler.HandleError(new Exceptions.InvalidRolloutException("Provided rollout is not in datafile."));
             return new Rollout();
         }
+
+        /// <summary>
+        /// Get the rollout rule from the ID
+        /// </summary>
+        /// <param name="rolloutRuleId">ID of the rollout rule</param>
+        /// <returns>Experiment Entity corresponding to the ID or a dummy entity if ID is invalid</returns>
+        public Experiment GetRolloutRuleFromId(string rolloutRuleId)
```

Comment:
This should no longer be needed since we don't need to look up the rule by ID anymore